aptos-move/framework/move-stdlib/sources/bit_vector.move

```diff
@@ -105,6 +109,9 @@ module std::bit_vector {
             };
         }
     }
+    spec shift_left {
+        pragma verify = false;
```

Comment:
Set to false because data invariant cannot be proved with inline function now. Will remove it once inline is supported

Reply:
I see. Could we leave a TODO to explain this a bit?